api/errors/errors.go

```diff
@@ -34,12 +34,18 @@ var ErrGetESDTTokens = errors.New("get esdt tokens for account error")
 // ErrGetESDTBalance signals an error in getting esdt balance for given address
 var ErrGetESDTBalance = errors.New("get esdt balance for account error")
 
+// ErrGetESDTNFTData signals an error in getting esdt nft data for given address, tokenID and nonce
+var ErrGetESDTNFTData = errors.New("get esdt nft data for account error")
```

Comment:
can you change please the text of this error?  ("cannot get esdt nft data for account" )

Reply:
followed all the error messages from this file